README.md

```diff
@@ -1,5 +1,75 @@
 daml-dit-arcade-index
 ====
 
-List of repositories included in the
-[project:DABL](https://projectdabl.com/) Arcade.
+This repository defines a list of packages that are included in the [Daml Hub](https://projectdabl.com/) Arcade. It consists of both **Sample Apps** and **Integrations**, whose locations are declared in the `repositories.yaml` index within this project.
+
+# Adding a Sample App
+
+Do you have a fully-fledged Daml application that runs on Daml Hub with a DAR model, triggers, custom UI, and other artifacts? Want to see it included as a one-click deployable Sample App? Read on.
+
+All Sample Apps (and Integrations) are consumed by Daml Hub in the form of [DIT files](https://github.com/digital-asset/daml-dit-api). In short, a DIT file is a single package that contains the metadata and deployable entities that make up your application.
+
+There are 5 straightforward tasks to getting your application ready and published on Daml Hub's arcade:
+
+1. [Configure](#Configure)
+2. [Build](#Build)
+3. [Release](#Release)
+4. [List](#List)
+5. [Update](#Update)
+
+> _Note_: Digital Asset reserves the right to make the ultimate determination of officially including your application in the arcade. However, an application DIT file can still be manually uploaded by any user to a Daml Hub ledger -- outside of the arcade -- so it may still prove a useful way of packaging, testing, and/or distributing non-official Daml Hub apps, depending on your use case.
+
+## Configure
+
+DIT packages are configured with a `dabl-meta.yaml` file in the root directory of your application.
+
+At a minimum, this file needs a top-level `catalog` key with the following items:
+
+```yaml
+catalog:
+    name: dabl-chat               # Machine-readable name
+    version: 0.2.2                # Semver version tag
+    short_description: DABL Chat  # Human readable name
+    description: A straightforward web chat app.
+    author: Digital Asset (Switzerland) GmbH
+    url: https://github.com/digital-asset/dablchat  # Link to source code
+    license: Apache-2.0
+    tags: [dabl-sample-app, application]  # These tags tell Daml Hub whether this package is an app or an integration
+    icon_file: dabl-chat.png      # File name for app icon
```

Comment:
Are any of these optional? I'd rather the example have a minimum required set of metadata

Reply:
You should link over to `daml_dit_api`, so readers can get to a full description of the structure.

https://github.com/digital-asset/daml-dit-api/blob/master/daml_dit_api/package_metadata.py#L35

Reply:
Most of the fields are optional, but that's only because of historical files that do not contain them. 

We need to encourage the catalog to be as fully populated as possible, so I (strongly) suggest we do not restrict the example to the minimum.